src/lib/battery/battery.cpp

```diff
@@ -113,7 +113,7 @@ void Battery::reset()
 	// TODO: check if it is sane to reset warning to NONE
 	_battery_status.warning = battery_status_s::BATTERY_WARNING_NONE;
 	_battery_status.connected = false;
-	_battery_status.capacity = _params.capacity;
```

Comment:
But now you're assigning NAN to an uint16?

Reply:
Gosh what am I thinking. :facepalm: